fe/fe-core/src/main/java/org/apache/doris/nereids/properties/DistributionSpecHash.java

```diff
@@ -54,6 +54,8 @@ public class DistributionSpecHash extends DistributionSpec {
     private final Set<Long> partitionIds;
     private final long selectedIndexId;
 
+    private final StorageBucketHashType storageBucketHashType;
```

Comment:
should update hashCode and equals function

Reply:
oh.. silly miss, I'll update